DOCS-3470 added instructions for approving and deleting access (#304)
mulesoft/docs-api-manager: modules/ROOT/pages/manage-client-apps-latest-task.adoc

= Approve or Reject Access Requests 
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords: grant access, api, contracts

After the user xref:exchange::to-request-access.adoc[requests access] to your API instance from Anypoint Exchange, as the Organization Administrator, you can manually approve or reject requests from API Manager if you have configured the SLA tier for the application for manual approvals.

To approve or reject requests to access your API:

. In API Manager, click *API Administration*.
. Click the name of the API instance for which you are requesting access. 

> . Click the `<api version>:<api instance>` link, for example, `v1:16046365`. 

Reviewer: This is not always `<api version>:<api instance>`, if the API Instance has a Label, then the label is shown.

Author: Done! Thanks!

> = Approve or Reject Access to Requests 

Reviewer 2: ```suggestion
= Approve or Reject Access Requests 
```
The "access" is not "access to requests"; it is access to what the request is asking access on behalf of.
The object of the request is access to something else.

Author: Done! Thanks!

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords: grant access, api, contracts

> After the user xref:exchange::to-request-access.adoc[requests access] to your API instance from Exchange, you can manually approve or reject a request from API Manager if you have configured the SLA tier for the application for manual approvals.

Reviewer 2: This text now switches from the audience being the user requesting access to the audience being the API instance owner. Clarify who the audience is for this topic set and what that audience's goal is: someone requesting access or someone granting access.

Author: Done! Thanks!

> After the user xref:exchange::to-request-access.adoc[requests access] to your API instance from Exchange, you can approve or reject a request from API Manager.

Reviewer: Only if manual approval. Otherwise, Contract will have been granted and only rejection is allowed.

Author: Done! Thanks!

> To approve or reject requests for accessing an API:

Reviewer 2: ```suggestion
To approve or reject requests to access your API:
```

Author: Done! Thanks!

. In API Manager, click *API Administration*.

> . Click the API Name of the API instance that you need to access. 

Reviewer 2: ```suggestion
. Click the name of the API instance for which access is being requested. 
```
Once again, please verify who or what is requesting access, what is being accessed, and who or what is approving or denying the request for access.

Author: Done! Thanks!

> . Click the `<api version>:<api instance>` link, for example, "v1:16046365". 

Reviewer 2: ```suggestion
. Click the `<api version>:<api instance>` link, for example, `v1:16046365`. 
```

Author: Done! Thanks!

> . In API Manager, click *Client Applications*.

Reviewer: Not exactly. This line should be `In API Manager, click in *API Administration*`.

The rest of the instructions are ok.

*Client Applications* page it can also be used to view contracts. But it starts from a Client Application instead of starting the journey based on a API Instance.

Author: Done! Thanks!

. From the configuration page for the API instance, click *Contracts* from the left menu. 
+
The request appears:
+
image::approve-requests-api-manager.png[Approving Requests,120%,110%]
. To approve the request, click *Approve*.

> . Alternatively, you can click *Reject* if you do not want to give access to your API instance to the requesting application.

Reviewer 2: ```suggestion
. Click either *Approve* or *Reject* to approve or reject the access request, respectively.
```

Author: Done! Thanks!
